<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectorAssemblyWidget</class>
 <widget class="QWidget" name="ProjectorAssemblyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProjectorAssemblyWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Available Extensions</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Pipeline</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" rowspan="3">
    <widget class="ExtensionConfigWidget" name="widget" native="true"/>
   </item>
   <item row="2" column="0" rowspan="2">
    <widget class="QListWidget" name="extensionList">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QListWidget" name="pipelineList_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QListWidget" name="pipelineList">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QTextBrowser" name="codeViewer">
     <property name="font">
      <font>
       <family>Liberation Mono</family>
      </font>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ExtensionConfigWidget</class>
   <extends>QWidget</extends>
   <header>gui/widgets/projectorassemblywidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>updateViewer()</slot>
 </slots>
</ui>
